core/src/main/java/org/infinispan/interceptors/impl/RetryingEntryWrappingInterceptor.java

```diff
@@ -33,7 +36,12 @@
    private final InvocationExceptionFunction<WriteCommand> handleManyWriteReturn = this::handleManyWriteReturn;
 
    @Override
-   protected Object setSkipRemoteGetsAndInvokeNextForDataCommand(InvocationContext ctx, DataWriteCommand command) {
+   protected Object setSkipRemoteGetsAndInvokeNextForDataCommand(InvocationContext ctx, DataWriteCommand command,
+         CompletionStage<Void> delay) {
+      if (!CompletionStages.isCompletedSuccessfully(delay)) {
```

Comment:
`BiasedEntryWrappingInterceptor` is checking if the `delay == null`. Is it missing here?

Reply:
The stage is always non null, the `BiasedEntryWrappingInterceptor` should have checked it if it was completed instead to avoid the additional allocation.

Reply:
:+1: